src/main/java/plugily/projects/buildbattle/handlers/themes/BBTheme.java

```diff
@@ -22,22 +22,14 @@
 
 public class BBTheme {
 
-  private String theme = "";
-  private final Difficulty difficulty;
+  public final String theme;
+  public final Difficulty difficulty;
 
   public BBTheme(String theme, Difficulty difficulty) {
     this.theme = theme == null ? "" : theme;
     this.difficulty = difficulty;
   }
 
-  public String getTheme() {
-    return theme;
-  }
-
-  public Difficulty getDifficulty() {
-    return difficulty;
-  }
-
```

Comment:
as it looks weird refering to that method without get we should get that again and change on arena getCurrentTheme to getCurrentBBTheme which then can do getTheme 
Would solve the miss calling of the old code.